example/spec/parallel/lzardsBackup/LzardsBackupSpec.js

```diff
@@ -0,0 +1,151 @@
+        FunctionName,
+      }).promise();
+
+      // Create the collection
+      collection = await createCollection(
+        prefix,
+        {
+          files: [
+            {
+              bucket: 'protected',
+              regex: '^[^.]+\.jpg$',
+              lzards: { backup: true },
+              sampleFileName: 'asdf.jpg',
+            },
+            {
+              bucket: 'protected',
+              regex: '^[^.]+\.dat$',
+              sampleFileName: 'asdf.dat',
+            },
+          ],
+        }
+      );
+
+      const Payload = JSON.stringify({
+        cma: {
+          ReplaceConfig: {
+            Path: '$.payload',
+            TargetPath: '$.payload',
+          },
+          task_config: {
+            cumulus_message: {
+              outputs: [
+                {
+                  source: '{$.originalPayload}',
+                  destination: '{$.payload}',
+                },
+                {
+                  source: '{$.backupResults}',
+                  destination: '{$.meta.backupStatus}',
+                },
+              ],
+            },
+          },
+          event: {
+            cumulus_meta: {
+              system_bucket: config.bucket,
+            },
+            meta: {
+              buckets: config.buckets,
+              collection,
+              provider,
+              stack: config.stackName,
+            },
+            payload: {
+              granules: [
+                {
+                  granuleId: 'FakeGranule2',
+                  dataType: collection.name,
+                  version: collection.version,
+                  files: [
+                    {
+                      name: 'testGranule.jpg',
+                      filename: `s3://${ingestPath}/testGranule.jpg`,
+                      checksumType: 'md5',
+                      checksum: '5799f9560b232baf54337d334179caa0',
+                    },
+                    {
+                      name: 'testGranule.dat',
+                      filename: `s3://${ingestPath}/testGranule.dat`,
+                      checksumType: 'md5',
+                      checksum: '39a870a194a787550b6b5d1f49629236',
+                    },
+                  ],
+                },
+              ],
+            },
+          },
+        },
+      });
+
+      lzardsBackupOutput = await pTimeout(
+        lambda().invoke({ FunctionName, Payload }).promise(),
+        (functionConfig.Timeout + 10) * 1000
+      );
+    } catch (error) {
+      beforeAllFailed = true;
+      throw error;
+    }
+  });
+
+  it('succeeds', () => {
+    if (beforeAllFailed) fail('beforeAll() failed');
+    else {
+      expect(lzardsBackupOutput.FunctionError).toBe(undefined);
+    }
+  });
+
+  it('has the expected backup information', () => {
+    const backupStatus = JSON.parse(lzardsBackupOutput.Payload).meta.backupStatus;
+    expect(backupStatus[0].status).toBe('COMPLETED');
+    expect(backupStatus[0].statusCode).toBe(201);
```

Comment:
>  It seems like these checks could be added to this test and this test could be removed?

If you mean the checks in this test could be added to the IngestGranuleSuccessSpec, you're not wrong. 

That said, the purposes of the two tests are different - IngestGranuleSuccessSpec is setup (or should be) to test the integration of our 'full' ingest chain.    As such workflow integration tests are showing that, in a typical configuration, the component works in interop with provided and documented components/configuration.

This test is explicitly to test the isolated component outside of a workflow.     As we add functionality, options, etc, we can expand this test to cover use cases that warrant integration testing explicitly without setting up workflows. 

If we're interested in de-duplicating this, I'd argue that we should remove it from the IngestSuccessSpec in favor of isolated component testing, but I think I see the value in showing our cookbooks/typical configuration all works together as well.

Reply:
I can see the argument, though if we don't yet need the isolated component test I somewhat question adding it. Regardless, since this test invokes a lambda manually, it's pretty fast so it's not a big deal